webrender/src/platform/windows/font.rs

```diff
@@ -214,6 +192,16 @@ impl FontContext {
                                          0.0, 0.0)
     }
 
+    pub fn get_glyph_index(&mut self, font_key: FontKey, ch: char) -> Option<u32> {
+        let face = self.fonts.get(&font_key).unwrap();
+        let indices = face.get_glyph_indices(&[ch as u32]);
+        if indices.len() == 1 {
```

Comment:
nit: if the length can't be higher than `1`, we could use `first().map(..)` here